### ISSUE TITLE
`download-folder-button` missing from commits and tags

### ISSUE THREAD

**Issue by hnynf** [refined-github/refined-github#7835]
<img width="1842" alt="image" src="https://github.com/user-attachments/assets/e9d65658-91ff-4d53-913b-813f25d6f836">

The token has been added.

**Comment by kovsu** [comment]
https://github.com/refined-github/refined-github/blob/863531f01eaeccd9d73cdee77f3d8b4de6787670/source/features/download-folder-button.tsx#L18

The download button depends on `a[aria-keyshortcuts="d"]`. Here, it doesn't have that button.

**Comment by fregante** [comment]
Next time please use the bug report form when something doesn't work.

**Comment by fregante** [comment]
The reason is that tags and commits don't have the "Delete directory" button:

https://github.com/sindresorhus/eslint-plugin-unicorn/tree/db1a6e05b00df3b62d55a110d48f041cabeb61a0/configs

But it works on branches:

https://github.com/sindresorhus/eslint-plugin-unicorn/tree/main/configs

**Comment by Banaanae** [comment]
Also missing from archived repositories
